README.md

```diff
@@ -19,11 +31,11 @@ import { querySelector, querySelectorAll } from 'kagekiri'
 
 // query the document
 const elements = querySelectorAll('.container button')
-const element = querySelector('.container button')
+const elements = getElementsByClassName('.container button')
 
 // or a specific element
 const elements = querySelectorAll('button', otherElement)
-const element = querySelector('button', otherElement)
+const elements = getElementsByClassName('button', otherElement)
```

Comment:
I'd prefer not to change these examples. It's much less impressive to show off `getElementsByClassName` because it's more trivial to implement than a query selector engine.

The reason for class queries is just because I wanted the example selectors to be simple.

Reply:
Sure thing! Will revert.